app/integration_action.go

```diff
@@ -141,15 +144,18 @@ func (a *App) DoPostActionWithCookie(postId, actionId, userId, selectedOption st
 
 	if response.Update != nil {
 		response.Update.Id = postId
-		response.Update.AddProp("from_webhook", "true")
```

Comment:
Why did this "from_webhook" addition exist?

Reply:
This was in the original PR by https://github.com/mattermost/mattermost-server/pull/7274, `interactive_action.go` is a new-ish file. Really can't speak to the reasons other than maybe it made sense at the time?

Reply:
Why are you removing it?

Reply:
@hanzei requested

Reply:
@hanzei why was this requested? Is this to enable bot accounts use interactive dialogs, or actions (buttons/menus)?

What is the consequence for removing this setting for existing integrations?

Reply:
Before this change using a message buttons/menu would reset the profile picture.

I can't think of a consequence for existing integrations, as this prop is always set anyway if you create a post via an integration.

Reply:
@hanzei I see. So to confirm: https://mattermost.atlassian.net/browse/MM-14795 is about fixing a bug where an action (via button or menu) resets the profile picture of the post?

If there is no consequence, does that mean `from_webhook` wasn't used?

Reply:
> So to confirm: mattermost.atlassian.net/browse/MM-14795 is about fixing a bug where an action (via button or menu) resets the profile picture of the post?

Yes.

> If there is no consequence, does that mean `from_webhook` wasn't used?

No. `from_webhook` is set when creating a post via an integration. This is happens. For some reason every actions _also_ added `from_webhook`. And this PR fixes this.

Reply:
Updated ticket title as it wasn't clear from the ticket what issue was been fixes.

Since there's no consequence, this is a low risk change that fixes a bug. Resolving.

Thanks Hanzei!